### PR TITLE
Fix bug with multi-intents

### PR DESCRIPTION
**Pull request by tabergma**
**Proposed changes**:
- Training with multi-intents using `CountVectorsFeaturizer` and `EmbeddingIntentClassifer` is working again.

closes https://github.com/RasaHQ/rasa/issues/5162

**Status (please check what you already did)**:
- [x] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: data/examples/rasa/demo-rasa-multi-intent.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+## intent:affirm
```
Comment (Ghostvv) [inline data/examples/rasa/demo-rasa-multi-intent.md]: in the test `"intent_split_symbol": "+"` but there is no such intent in this file

Comment (tabergma) [reply data/examples/rasa/demo-rasa-multi-intent.md]: äh... I changed the chitchat intents, strange, will do the changes again

Comment (Ghostvv) [reply data/examples/rasa/demo-rasa-multi-intent.md]: `/` in chitchat intents is a special character to indicate response selection

Comment (tabergma) [reply data/examples/rasa/demo-rasa-multi-intent.md]: yeah, but we do not test response selection in this test and the data file is only used in this one test, so should be fine to replace it

**Pull request merged**